cylc/flow/platforms.py

```diff
@@ -30,6 +30,10 @@
     ('job', 'batch submit command template', [None])
 )
 
+# Regex to check whether a string is a command
```

Comment:
I realized that I probably wanted to keep these in a single, centralized location.